README.rst

```diff
@@ -57,7 +57,7 @@ CherryPy is a pythonic, object-oriented HTTP framework.
 
 1. It allows building web applications in much the same way one would
    build any other object-oriented program.
-2. This design results in less and more readable code being developed faster.
+2. This design results in short and more readable code being developed faster.
```

Comment:
Could you explain the motivation behind this change?

Reply:
There's ambiguity with "less". It could be read as "less readable and more readable code". The phrasing could be improved.

```suggestion
2. This design results in more concise and readable code developed faster.
```

Reply:
@ps-19 FYI you may accept Jason's suggestion by clicking the `Commit suggestion` button above.